README.md

```diff
@@ -348,7 +348,7 @@ To learn more about columnar storage, check out the [columnar storage README](ht
 
 ## Documentation
 
-If you’re ready to get started with Citus or want to know more, we recommend reading the [Citus open source documentation](https://docs.citusdata.com/en/stable/). Or, if you are using Citus on Azure, then the [Hyperscale (Citus) documentation](https://docs.microsoft.com/azure/postgresql/hyperscale/) is online and available as part of the Azure Database for PostgreSQL docs.
+If you’re ready to get started with Citus or want to know more, we recommend reading the [Citus open source documentation](https://docs.citusdata.com/en/stable/). Or, if you are using Citus on Azure, then the [Azure Cosmos DB for PostgreSQL](https://learn.microsoft.com/en-us/azure/cosmos-db/postgresql/introduction) is online and available as part of the Azure Cosmos DB for PostgreSQL docs.
```

Comment:
1) Please remove "/en-us" from the docs/learn URL, so that the URL will redirect to whichever locale is set on the reader's system
2) Need to remove the "deployment option" term too
3) I think the tricky part is the name of our managed service used to have 2 parts: Hyperscale (Citus) and Azure Database for PostgreSQL. Above, we've replaced BOTH parts with "Azure Cosmos DB for PostgreSQL" which is creating a repetition. Can you please remove the repetition, in both the 1st sentence on this line, and in the 3rd sentence too!